### ISSUE TITLE
wl-folder-goto-folder sets this-command nil before calling wl-summary-read-folder

### ISSUE THREAD

**Issue by yashi** [wanderlust/wanderlust#109]
Adding `wl-folder-goto-folder` to `ido-migemo-exclude-command-list` does not make ido-migemo to ignore the function.  The reason is that the function shadows `this-command` with `nil` in ab813f78fe.  

Fixing this particular problem is easy. Just `(let ((this-command this-command)))` however I'm not sure what was the original reason behind this change.  The commit reads:

```
	* wl-folder.el (wl-folder-goto-folder-subr): Keep value of
	`this-command' so that `wl-folder-goto-folder' contained in
	`wl-summary-keep-cursor-command' work.
```

So, keeping the original value is good enough, or does it need to be `nil`?
Anyone know the reason or how to test the original problem?

**Comment by ikazuhiro** [comment]
According to ML's log, the change was committed to fix the problem `wl-summary-keep-cursor-command` had no effect on `wl-folder-goto-folder` function.  As far as I confirmed, there is no code which expect `this-command`'s value is `nil`.  So I think let-binding `this-command` to `nil` was the result of simply choosing shorter code.
I will commit your fix later.

**Comment by yashi** [comment]
Would you mind to give me the link to ML's log?  I can put it in my commit log and send you a pull request, if that helps.

**Comment by yashi** [comment]
Oops.  You have already fixed this issue by 4e49bbbdc.  Thanks. :smile: 
Closed.

**Comment by ikazuhiro** [comment]
> Would you mind to give me the link to ML's log?

I couldn't find old log online.  I searched from my local archive.  If you still need related log, please let me know with your e-mail address.
